### PR TITLE
Backend Integration: Login To Your Account

### PR DESCRIPTION
**Pull request by LamboCreeper**
This allows users to log into their accounts. Currently, there is no way to create an account except from using a tool like Postman.

### CODE REVIEW

**Comment by LamboCreeper** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- src/components/pages/LoginPage.js  1
- src/components/organisms/LogInForm.js  3
- src/services/AuthenticationService.js  1
- src/components/molecules/Navigation.js  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5714005&bid=18699915)

File: src/css/pages/loginpage.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.login-area {
+    padding: 100px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
```
Comment (LamboCreeper) [inline src/css/pages/loginpage.css:6]: Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5714005)

**Comment by LamboCreeper** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/components/molecules/Navigation.js  3
- src/components/pages/LoginPage.js  1
- src/services/AuthenticationService.js  1
- src/components/organisms/LogInForm.js  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5714005&bid=18699915)

**Comment by LamboCreeper** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/components/organisms/LogInForm.js  3
- src/components/molecules/Navigation.js  3
- src/services/AuthenticationService.js  1
- src/components/pages/LoginPage.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5714005&bid=18699915)

**Review by Jason2605 — APPROVED** [review]
LGTM

**Comment by LamboCreeper** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/components/molecules/Navigation.js  3
- src/components/pages/LoginPage.js  1
- src/components/organisms/LogInForm.js  3
- src/services/AuthenticationService.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/codesupport/website-frontend/pullRequest?prid=5714005&bid=18699915)

**Pull request merged**